Providers/SelectedListManager.tscn

```diff
@@ -17,6 +19,16 @@ func apply_values_to_selected_type(parameter_name: String, parameter_value):
 # Function to check the list for type and removes if it exists already
 func toggle_selection(placemark: GenericPlacemark, data: Object):
 	var entry = {\"placemark\": placemark, \"data\": data}
+	
+	if placemark is EnemyPlacemark:
+		enemy_type = true
+		
+	if placemark is GatheringItemPlacemark:
+		item_type = true
+		
+	if enemy_type and item_type:
+		clear_list()
```

Comment:
clear_list should set both _type variables to false otherwise they never ever again go back to being false